packages/modules/create-itwin-viewer/extensions/web/intermediate.ts

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "@itwin/web-viewer-react";
```

Comment:
I believe that @aruniverse mentioned this already, but this one is not needed

Reply:
done, changed to the viewer navigation tools